modules/images/dominant-color/load.php

```diff
@@ -117,7 +117,7 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, $context, $
 
```

Comment:
Not strictly related to this fix, but there is a problem in this function, which is exactly similar to the bug flagged in https://core.trac.wordpress.org/ticket/56969: We must avoid adding any of these attributes to `img` tags which don't use regular double quotes.

For this reason, let's add a check for `' src="'` to the top of this function. If that is not part of the `img` string, we should bail early and not even run the filter, similar to how it's done for lazy-loading. See https://core.trac.wordpress.org/ticket/56969#comment:16.